packages/core-sass/src/stylesheets/utilities/_color_variables.scss

```diff
@@ -0,0 +1,82 @@
+/* BRAND COLORS */
+$hds-brand-color-coat-of-arms-blue: #0072c6;
+$hds-brand-color-coat-of-arms-blue-light-20: #cce2f3;
+$hds-brand-color-coat-of-arms-blue-light-50: #80b8e2;
+$hds-brand-color-coat-of-arms-blue-dark-50: #000098;
+
+$hds-brand-color-coat-of-arms-gold: #c2a251;
+$hds-brand-color-coat-of-arms-gold-light-20: #f2ecdc;
+$hds-brand-color-coat-of-arms-gold-light-50: #e0d0a8;
```

Comment:
This will make duplicated var, cause HDS defined vars in css vars also
--hds--black-80: #black80Hex
$--hds--black-80: #black80Hex

Not single source of truth

Reply:
well, this is something that is not bad actually, if we offer sass partials, the sass variables should be exposable. I used bootstrap-sass as a reference.